src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private double firstLeg;
+    private double secondLeg;
+
+    public RightTriangle(String color, double firstLeg, double secondLeg) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (firstLeg * secondLeg) / 2;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Right triangle: firstLeg, secondLeg:" + firstLeg + "," + secondLeg
+                + "and area is: " + calculateArea());
```

Comment:
The output string in the draw method is missing spaces and punctuation for better readability. Consider reformatting the string to: 'Right triangle: firstLeg = ' + firstLeg + ', secondLeg = ' + secondLeg + ', area = ' + calculateArea() + ', color = ' + getColor()